TODOList.md

```diff
@@ -0,0 +1,14 @@
+# Google PACO Source Code List of TODO
```

Comment:
I would like to translate the application for Swift. Can I take part in the development of the next version of the application?